ironfish-cli/src/commands/accounts/pay.ts

```diff
@@ -159,10 +164,10 @@ export class Pay extends IronfishCommand {
       this.log(`
 You are about to send:
 ${displayIronAmountWithCurrency(
-  amount,
+  Number(amount),
   true,
 )} plus a transaction fee of ${displayIronAmountWithCurrency(
-        fee,
+        Number(fee),
```

Comment:
This is a good fix, but it actually reveals that the types for these 2 in this whole file are broken. I can fix them in another PR, or do you want to fix them in this PR? For one, they are strings that are just type-casted (not value casted) to numbers when they are actually strings the whole time above.